src/main/java/gregtech/api/GTValues.java

```diff
@@ -55,7 +55,7 @@
     /**
      * The Voltage Tiers adjusted for cable loss.
      */
-    public static final long[] VA = new long[]{7, 30, 120, 480, 1920, 7680, 30720, 122880, 491520, 1966080, 7864320, 31457280, 125829120, 503316480, 2013265920};
+    public static final int[] VA = new int[]{7, 30, 120, 480, 1920, 7680, 30720, 122880, 491520, 1966080, 7864320, 31457280, 125829120, 503316480, 2013265920};
```

Comment:
Would be helpful to add that these are for recipes to in the comment, not just cable loss.